docs/monitoring/README.md

```diff
@@ -0,0 +1,49 @@
+# Monitoring
```

Comment:
What about adding the overview diagram from https://github.com/gardener/gardener/issues/2946#issuecomment-822266893?